terraform/dev/eks-cluster.tf

```diff
@@ -0,0 +1,38 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "19.0.4"
+
+  cluster_name    = local.cluster_name
+  cluster_version = "1.24"
+
+  vpc_id                         = var.vpc_id
+  subnet_ids                     = ["subnet-a3f202fa", "subnet-fc843999", "subnet-0fc91a78"]
+  cluster_endpoint_public_access = true
+  cluster_security_group_id      = var.security_group_id
+  aws_auth_users = [
+    {
+      userarn  = "arn:aws:iam::364159549467:user/liam"
+      username = "liam"
+      groups   = ["system:masters"]
+    }
+  ]
+
+  eks_managed_node_group_defaults = {
+    ami_type = "AL2_x86_64"
+
+  }
+
+  eks_managed_node_groups = {
+    one = {
+      name = "node-group-1"
+
+      vpc_security_group_ids = [var.security_group_id]
+
+      instance_types = ["t3.small"]
```

Comment:
We might want these instances to be larger when stela starts to have significant workload, but updating them to larger instances should be simple (no worse in impact that our current infrastructure deploys).